File: code-coaches/easy/hovercraft/hovercraft.rb
def hovercraft_profit_status(units_sold)
  sell_price = 3_000_000
  cost_price = 2_000_000
  insurance_price = 1_000_000

  units_built = 10
  expenses = (units_built * cost_price) + insurance_price
  revenue = units_sold * sell_price

  if revenue > expenses
    return "Profit"
  elsif revenue < expenses
    return "Loss"
  else
    return "Broke Even"
  end
end

def main
  units_sold = gets.chomp.to_i
  puts hovercraft_profit_status(units_sold)
end

if __FILE__ == $0
  main
end

File: code-coaches/easy/hovercraft/hovercraft.c
#include <stdio.h>

int main()
{
    int unitsSold;
    scanf("%d", &unitsSold);

    int sellPrice = 3000000;
    int costPrice = 2000000;
    int insurancePrice = 1000000;

    int unitsBuilt = 10;
    int expenses = (unitsBuilt * costPrice) + insurancePrice;
    int benefit = unitsSold * sellPrice;

    if (benefit > expenses)
    {
        printf("Profit");
    }
    else if (benefit < expenses)
    {
        printf("Loss");
    }
    else
    {
        printf("Broke Even");
    }

    return 0;
}
